src/Jeddeloh__ApolloClient.re

```diff
@@ -10,25 +10,23 @@ module Bindings = {
 
 module GraphQL_PPX = {
   module ExtendMutation = ApolloClient__React_Hooks_UseMutation.Extend;
-  module ExtendMutationNoRequiredVariables = ApolloClient__React_Hooks_UseMutation.ExtendNoRequiredVariables;
+  module ExtendMutationNoRequiredVariables = ApolloClient__React_Hooks_UseMutation.Extend;
```

Comment:
You can even remove the no-required-variable configuration (it's just extra configuration if you need it). Less configuration!

Reply:
Nice!